com.pokemmo.PokeMMO.yaml

```diff
@@ -0,0 +1,80 @@
+app-id: com.pokemmo.PokeMMO
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk
+command: pokemmo.sh
+tags:
+  - proprietary
+finish-args:
+- --socket=x11
+- --share=ipc
+- --socket=wayland
+- --socket=pulseaudio
+- --device=all
+- --share=network
+- --env=PATH=/app/jre/bin:/app/bin:/usr/bin
+
+modules:
+  - shared-modules/libdecor/libdecor-0.2.0.json
+  - name: glfw
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=RelWithDebInfo
+      - -DBUILD_SHARED_LIBS:BOOL=ON
+      - -DGLFW_USE_WAYLAND=ON
+    sources:
+      - type: git
+        url: https://github.com/glfw/glfw.git
+        commit: 3eaf1255b29fdf5c2895856c7be7d7185ef2b241
+    cleanup:
+      - /include
+      - /lib/cmake
+      - /lib/pkgconfig
+      - /app/share/doc/GLFW/
+
+  - name : openjdk
+    buildsystem : simple
+    build-commands :
+      - /usr/lib/sdk/openjdk/install.sh
+
+  - name: pokemmo
+    buildsystem: simple
+    build-commands:
+      - install -D pokemmo.sh /app/bin/pokemmo.sh
+      - install -Dm 755 apply_extra /app/bin/apply_extra
+      # Icon
+      - install -Dm644 16x16.png /app/share/icons/hicolor/16x16/apps/com.pokemmo.PokeMMO.png
+      - install -Dm644 32x32.png /app/share/icons/hicolor/32x32/apps/com.pokemmo.PokeMMO.png
+      - install -Dm644 128x128.png /app/share/icons/hicolor/128x128/apps/com.pokemmo.PokeMMO.png
+      # Desktop entry
+      - install -Dp -m 644 com.pokemmo.PokeMMO.desktop /app/share/applications/com.pokemmo.PokeMMO.desktop
+      # Placing appdata
+      - install -Dm644 -t /app/share/metainfo com.pokemmo.PokeMMO.metainfo.xml
+    sources:
+      - type: extra-data
+        filename: PokeMMO-Client.zip
+        url: https://dl.pokemmo.com/download/PokeMMO-Client.zip
```

Comment:
does this work on aarch64?

Reply:
unversioned URL, this will not go well.

Reply:
> does this work on aarch64?

I wouldn't even know how to test it, it is written in java so maybe it works? As soon as someone notifies me that it does not work I will remove the arm version

Reply:
> > does this work on aarch64?
> 
> I wouldn't even know how to test it, it is written in java so maybe it works? As soon as someone notifies me that it does not work I will remove the arm version

nevermind. I tested  it with a linux phone, it works